### PR TITLE
feat(i18n): swedish translations

### PR DESCRIPTION
**Pull request by jrc2139**
## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [ ] All tests passing
- [x] No breaking changes

### CODE REVIEW

**Comment by ueman** [comment]
Awesome thanks @jrc2139 ! Is there a reason, that this PR is still marked as a draft?

**Comment by jrc2139** [comment]
> Awesome thanks @jrc2139 ! Is there a reason, that this PR is still marked as a draft?

had it set to draf to confirmed it worked.  good to go!

File: feedback/lib/src/l18n/translation.dart [hunk 1]
```diff
@@ -211,6 +229,7 @@ class GlobalFeedbackLocalizationsDelegate
     const Locale('fr'): const FrFeedbackLocalizations(),
     const Locale('ar'): const ArFeedbackLocalizations(),
     const Locale('ru'): const RuFeedbackLocalizations(),
+    const Locale('sv'): const RuFeedbackLocalizations(),
```
Comment (ueman) [inline feedback/lib/src/l18n/translation.dart:232]:
```suggestion
    const Locale('sv'): const SvFeedbackLocalizations(),
```

Comment (jrc2139) [reply feedback/lib/src/l18n/translation.dart]: 🤦 sry about that.. i was trying to run it by using git urls in my pubspec but the feedback_sentry plugin was freaking out about it.  i ended up editing the cached source files and forgot to update this PR

Comment (ueman) [reply feedback/lib/src/l18n/translation.dart]: No worries :D

**Pull request merged**